doc/man/restic-generate.1

```diff
@@ -3,7 +3,7 @@
 
 .SH NAME
 .PP
-restic-generate - Generate manual pages and auto-completion files (bash, fish, zsh)
+restic-generate - Generate manual pages and auto-completion files (bash, fish, zsh, powershell)
```

Comment:
This entire file is generated as part of the release build process, so please remove all changes to this file :)